src/pages/Home/components/Map/components/MobileOptions/index.tsx

```diff
@@ -1,66 +1,74 @@
 import { ReactComponent as LensIcon } from 'assets/svg/home/lens.svg';
-import { ReactComponent as GroupIcon } from 'assets/svg/home/group.svg';
+import { ReactComponent as StoreFrontIcon } from 'assets/svg/home/storefront.svg';
 import { ReactComponent as BookMarkIcon } from 'assets/svg/home/bookmark.svg';
-import { ReactComponent as VerticalDot } from 'assets/svg/home/verticalDot.svg';
+import { ReactComponent as GroupIcon } from 'assets/svg/home/group.svg';
+import { ReactComponent as VerticalDot } from 'assets/svg/home/verticaldot.svg';
 import { Link } from 'react-router-dom';
-import cn from 'utils/ts/classNames';
-import { useState } from 'react';
 import useBooleanState from 'utils/hooks/useBooleanState';
+import { useFilterFriend, useFilterNearby, useFilterScrap } from 'store/filter';
+import cn from 'utils/ts/classNames';
 import styles from './MobileOptions.module.scss';
 
 export default function MobileOptions(): JSX.Element {
-  const [filter, , ,toggle] = useBooleanState(false);
-  const [selected, setSelected] = useState('');
-  const handleClick = (type: string) => {
-    setSelected(type);
-    // 각각의 함수 추가 필요
-  };
+  const [visible, , , toggle] = useBooleanState(true);
+  const { filterFriendState, setFilterFriend } = useFilterFriend();
+  const { filterScrapState, setFilterScrap } = useFilterScrap();
+  const { filterNearbyState, setFilterNearby } = useFilterNearby();
+
   return (
-    <div className={styles.options}>
-      <div className={styles.nav}>
-        <div>
-          <Link to="/search" className={styles.nav__search}>
-            검색어를 입력해주세요.
-            <LensIcon />
-          </Link>
-        </div>
-        <button
-          type="button"
-          className={cn({
-            [styles.nav__filter]: true,
-            [styles['nav__filter--clicked']]: selected !== '' && !filter,
-          })}
-          onClick={toggle}
-        >
-          <VerticalDot />
-        </button>
-        {filter && (
-        <div className={styles.nav__list}>
+    <div className={styles.nav}>
+      <Link to="/search" className={styles.nav__search}>
+        검색어를 입력해주세요.
+        <LensIcon />
+      </Link>
+      <button
+        type="button"
+        className={cn({
+          [styles.nav__icon]: true,
+          [styles['nav__icon--changed']]: filterNearbyState === 1 || filterScrapState === 1 || filterFriendState === 1,
```

Comment:
boolean이라면 아래처럼 써도 괜찮을 것 같은데, 서버에서 1, 0을 쓴다고 저희도 1, 0으로 코드를 짤 필욘 없을 것 같아요(그냥 가독성을 위해서니깐 굳이 적용은 안해도 될 것 같아요 그냥팁)
```suggestion
          [styles['nav__icon--changed']]: filterNearbyState || filterScrapState || filterFriendState,
```

Reply:
이 부분도 추후에 수정하는 것으로 하겠습니다!